src/main/java/lotto/application/LottoService.java

```diff
@@ -1,46 +1,29 @@
 package lotto.application;
 
-import java.util.List;
-import java.util.stream.Collectors;
-import java.util.stream.IntStream;
-
 import lotto.domain.LottoLotteries;
+import lotto.domain.LottoLotteriesFactory;
 import lotto.domain.LottoLottery;
-import lotto.domain.LottoNumber;
 import lotto.domain.LottoResults;
-import lotto.domain.Message;
 import lotto.domain.Money;
+import lotto.domain.RandomLottoNumberGenerator;
 import lotto.domain.WinLottoNumbers;
-import lotto.utils.RandomLottoNumberGenerator;
 
 public class LottoService {
+	private final RandomLottoNumberGenerator randomLottoNumberGenerator = new RandomLottoNumberGenerator();
 	private final LottoLotteries lottoLotteries;
 
 	public LottoService(Money money) {
```

Comment:
```suggestion
	public LottoService(Money money, LottoNumberGenerator lottoNumberGenerator) {
		this.lottoNumberGenerator = lottoNumberGenerator;
		this.lottoLotteries = buyMaxAutoLottoLotteries(money);
	}
```

의존성 주입을 사용해서 로또번호 생성 객체를 외부에서 주입 받는 형식으로 교체해보면 어떨까요

Reply:
generator를 어디서 생성할까 고민은 했었는데요. 유연성을 위해 외부에서 주입 받는 방식이 좋을것 같습니다. 감사합니다.